app/controllers/admin/pictures_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Admin::PicturesController < AdminController
-  before_action :set_picture, only: [:edit, :destroy, :update, :load_elements]
+  before_action :set_picture, only: [:show, :edit, :destroy, :update, :load_elements]
```

Comment:
__[rubocop]__
* Line is too long. [85/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)